Funcky/Monads/Lazy/Lazy.Core.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace Funcky.Monads.Lazy
+{
+    public static class Lazy
+    {
+        public static Lazy<TItem> Return<TItem>(Func<TItem> valueFactory)
```

Comment:
Is `Return` the right name for this? `Return` usually takes the monad's value, not a `Func`.
I'd call this `FromFunc` or `From` instead. What do you think?

Reply:
Yeah that is probably true, we have the same Problem on the Reader.

Reply:
Lazy.Return renamed: 4204256d
Reader-Monad uses the same pattern.